test/money_test.exs

```diff
@@ -339,6 +339,18 @@ defmodule MoneyTest do
     end
   end
 
+  test "to_string configuration with custom_display_options" do
+    try do
+      Application.put_env(:money, :custom_display_options,
+        EUR: %{symbol: "€", symbol_on_right: true, symbol_space: true, separator: ".", delimiter: ","}
```

Comment:
I'm not sure if here we should use the symbol or a boolean. Probably it would make more sense to use a boolean as we can set the symbol already in `:custom_currencies`.

Reply:
I have updated the code to accept a boolean in the symbol atom.